build.gradle

```diff
@@ -49,14 +51,22 @@ subprojects { subproject ->
 	apply plugin: 'idea'
 	apply plugin: 'jacoco'
 	apply plugin: 'checkstyle'
+	apply plugin: 'kotlin'
```

Comment:
Let's consider to add `kotlin-spring` as well?
https://github.com/spring-projects/spring-integration/pull/2432